ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -2202,10 +2243,78 @@ export const WorkspaceEdit = fp.flow(
               </FlexRow>
             </WorkspaceEditSection>
 
+            {/* AIAN section*/}
+            {askAboutAIAN && (
+              <WorkspaceEditSection
+                header={researchPurposeQuestions[9].header}
+                index='6.'
+                indent={true}
+                description={researchPurposeQuestions[9].description}
+                style={{ width: '72rem' }}
+              >
+                {/* RadioGroup: AIAN Research Type */}
+                <FlexColumn>
+                  <FlexRow>
+                    <div style={{ ...styles.header, marginRight: '0.6rem' }}>
+                      6.1
+                    </div>
+                    <div style={styles.header}>
+                      {researchPurposeQuestions[10].header}
+                    </div>
+                  </FlexRow>
+                  <div style={{ marginLeft: '1.35rem' }}>
+                    {Array.from(aianResearchTypeMap.keys()).map(
+                      (aianResearchTypeOption) => (
+                        <FlexRow style={{ paddingBottom: '0.9rem' }}>
```

Comment:
This should have a key prop, because it's an array element.  https://react.dev/learn/rendering-lists